docs/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: qtconsole_docs
 channels:
+  - astrofrog
```

Comment:
I don't think we need an extra channel here - pyqt is in the standard conda channel.